Signal/src/view controllers/CallViewController.swift

```diff
@@ -264,6 +480,19 @@ class CallViewController: UIViewController {
         }
     }
 
+    func allControls() -> [UIView] {
+        return incomingCallControls() + ongoingCallControls()
+    }
+
+    func incomingCallControls() -> [UIView] {
```

Comment:
It turns out the preferred syntax is to skip `get{}` unless there is also a `set()`.

```
    var incomingCallControls: [UIView] {
        return [ incomingTextMessageButton, acceptIncomingButton, declineIncomingButton, ]
    }
```